website/package.json

```diff
@@ -8,12 +8,13 @@
   },
   "dependencies": {
     "@hashicorp/mktg-global-styles": "2.1.0",
-    "@hashicorp/nextjs-scripts": "16.0.1",
+    "@hashicorp/nextjs-scripts": "^16.3.0",
```

Comment:
Bump to `nextjs-scripts` was required, to ensure `withProductMeta` was available.